models/maisi_ct_generative/docs/README.md

```diff
@@ -23,10 +23,6 @@ The inference requires:
 - Disk Memory: at least 21GB disk memory
 
 ### Execute inference
-The model weights can be downloaded with
-```
-python -m scripts.download_files
-```
```

Comment:
I am not sure how users can download the model weights if we remove this part.

Reply:
Hi @dongyang0122, users don't need download large files by themselves since when they downloading the bundle, all these large files have already been downloaded and inlcuded in the bundle.